images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -331,6 +331,18 @@ function Get-DotNetCoreSdkVersions {
     return $dotNetCoreSdkVersion
 }
 
+function Get-DotnetTools {
```

Comment:
We should add `$env:PATH = "/etc/skel/.dotnet/tools:$($env:PATH)"` in this function too.